src/backend/electron/fileHelper.ts

```diff
@@ -0,0 +1,15 @@
+import fs from "fs";
+import { moveFile } from "move-file";
+
+export async function writeFileSafely(
+  path: string,
+  data: string | NodeJS.ArrayBufferView,
+) {
+  // ファイル書き込みに失敗したときに設定が消えないように、tempファイル書き込み後上書き移動する
+  const temp_path = `${path}.tmp`;
+  await fs.promises.writeFile(temp_path, data);
+
+  await moveFile(temp_path, path, {
```

Comment:
あ、これ元コードで気づかなかったけどsnake_caseになってますね！
ちょっとついでにcamelCaseに直しちゃえると！！
```suggestion
  const tempPath = `${path}.tmp`;
  await fs.promises.writeFile(tempPath, data);

  await moveFile(tempPath, path, {
```